File: 215_Kth_Largest_Element_in_an_Array/solution.py
# class Solution:
#     def findKthLargest(self, nums: List[int], k: int) -> int:
#         sorted_nums = sorted(nums)
#         n = len(nums)
#         return sorted_nums[n - k]

class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        return sorted(nums)[-k]

File: 215_Kth_Largest_Element_in_an_Array/solution.c
int compareInts(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int findKthLargest(int* nums, int numsSize, int k) {
    qsort(nums, numsSize, sizeof(int), compareInts);
    return nums[numsSize - k];
}